ui/src/index.html

```diff
@@ -2,7 +2,8 @@
 <html class="no-js" ng-app="bassa">
   <head>
     <meta charset="utf-8">
-    <title>Bassa</title>
+    <!--title>Bassa</title-->
```

Comment:
Why did you comment the title ? Don't add commented code. 👍 Keep up the good work. 😄